core/frontend/src/DisplayStyleState.ts

```diff
@@ -221,24 +221,23 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
    * Set the display of the OpenStreetMap worldwide building layer in this display style by attaching or detaching the reality model displaying the buildings.
    * The OSM buildings are displayed from a reality model aggregated and served from Cesium ion.<(https://cesium.com/content/cesium-osm-buildings/>
    */
-  public setOSMBuildingDisplay(options: { onOff?: boolean }): boolean {
+  public setOSMBuildingDisplay(options: { onOff?: boolean, appearanceOverrides?: FeatureAppearance }): boolean {
```

Comment:
What should user expect to happen if they turn OSM buildings on with some appearance overrides, then turn them off, then back on without specifying overrides?

Reply:
I'm not sure -- If you detach a reality model and reattach then you lose appearance overrides.  Could add a property to just turn off display of any reality model without detaching but that may also introdude complication and confusion.

Reply:
The way it works now sounds right to me. But not evident without digging through the code to find out.